atd-vze/src/views/Crashes/CrashChange.js

```diff
@@ -601,241 +593,6 @@ function CrashChange(props) {
       });
   };
 
-  /**
-   * Returns true if the current field (key) is PII
-   * @param {string} recordType - The record type
-   * @param {string} key - The name of the field
-   * @returns {boolean}
-   */
-  const isFieldPII = (recordType, key) => {
```

Comment:
This caught my eye, but then I saw that the crash records don't have any fields in the PII config that this uses. 👍

Reply:
same, also curious why PII fields were removed? why wouldnt we want those changes updated?